scripts/embassy.ts

```diff
@@ -1,6 +1,5 @@
 export { setConfig } from "./procedures/setConfig.ts";
 export { getConfig } from "./procedures/getConfig.ts";
-export { properties } from "./procedures/properties.ts";
+// export { properties } from "./procedures/properties.ts";
```

Comment:
same question as above

Reply:
Properties were trying to load - but there aren't any